### PR TITLE
New cli.entrypoint instead of juju_verify.main

### PR DESCRIPTION
**Pull request by r-f-g**
- rename juju_verify.py to cli.py
 - move find_units and find_units_on_machine
 - add JujuVerifyError as logic error in juju-verify cli
 - remove fail function and use raise in each function and sys.exit(1)
   only in entrypoint
 - using MagicMock instead of Model in unit tests (error if current
   model is destroyed)

### CODE REVIEW

File: tests/unit/test_entrypoint.py [hunk 1]
```diff
@@ -14,23 +14,21 @@
 #
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see https://www.gnu.org/licenses/.
-"""Test suite for the entrypoint function and helpers."""
+"""Test suite for the cli function and helpers."""
```
Comment (auria) [inline tests/unit/test_entrypoint.py:17]: I think the function name is still "entrypoint". The module is named "cli", though.

**Review by auria — DISMISSED** [review]
minor suggestion, not a blocker.

+1.

File: tests/unit/test_entrypoint.py (hunk 1 above)
Comment (r-f-g) [reply tests/unit/test_entrypoint.py]: Thanks, I also renamed the file.

**Review by auria — APPROVED** [review]
+1

**Pull request merged**